### PR TITLE
Fix reporting test result :unknown

### PR DESCRIPTION
**Pull request by rajdeepv**
When the results are unknown, the format is not as expected. This fixes it

### CODE REVIEW

File: lib/parallel_cucumber/worker.rb [hunk 1]
```diff
@@ -291,18 +291,28 @@ def setup(env)
...
+    end
+
+    private
+
+    def unknown_result(tests)
+      res = {}
+      tests.each do |test|
+        res[test.to_sym] = { status: ::ParallelCucumber::Status::UNKNOWN }
+      end
+      res
```
Comment (bayandin) [inline lib/parallel_cucumber/worker.rb:315]:
```suggestion
      tests.map do |test, status|
        [test.to_sym, { status: ::ParallelCucumber::Status::UNKNOWN }].
      end.to_h
```
or, if you like
```suggestion
      res = tests.map do |test, status|
        [test.to_sym, { status: ::ParallelCucumber::Status::UNKNOWN }].
      end
      res.to_h
```

And also I would move `unknown_result` to `Helper::Cucumber` to make it clear why do we have this format

Comment (fr0l) [reply lib/parallel_cucumber/worker.rb:315]: ```status``` variable in the block is not used, so ```_status``` :-)

Comment (bayandin) [reply lib/parallel_cucumber/worker.rb:315]: > `status` variable in the block is not used, so `_status` :-)

It's not needed actually. Fixed the comment

Comment (rajdeepv) [reply lib/parallel_cucumber/worker.rb]: Moved

Comment (fr0l) [reply lib/parallel_cucumber/worker.rb]: but. yeah. the ```map``` variant is definitely a way to go

**Pull request merged**